packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcPreview.ts

```diff
@@ -73,20 +139,26 @@ export async function forceLightningLwcPreview(sourceUri: vscode.Uri) {
   }
 
   const fullUrl = `${DEV_SERVER_PREVIEW_ROUTE}/${componentName}`;
+  const isMobileEnabled = getWorkspaceSettings().get(mobileEnabledKey) || false;
+  if (!isMobileEnabled) {
+    await startServer(true, fullUrl, startTime);
```

Comment:
Minor: can we add a comment before this command, just stating something along the lines of "Go with the default Desktop experience if mobile is not enabled"?